README.md

```diff
@@ -153,6 +153,26 @@ $ vagrant up $YOUR_CHOICE
 $ vagrant ssh $YOUR_CHOICE
 ```
 
+### Nix
+
+There is experimental support for building and testing with [Nix flakes](https://nixos.wiki/wiki/Flakes).
+The quickstart follows but more detailed documentation can be found in [nix.md](docs/nix.md).
+
+First, install `nix` (see https://nixos.org/download.html):
+
+```
+$ sh <(curl -L https://nixos.org/nix/install) --daemon
```

Comment:
I'm not sure it's great to include the install command here, particularly as it ends up using high privilege at points. Probably best to link out to the Nix download page as you have above.

Reply:
Ack. Best to let the nix download page sell the mechanism.